src/binary/runBinary.ts

```diff
@@ -18,6 +18,8 @@ export default async function runBinary(
     : undefined;
   const args: string[] = [
     "--no-lsp=true",
+    "--tls_config",
+    `insecure=${tabnineExtensionProperties.ignoreCertificateErrors}`,
```

Comment:
this can't be merged before the binary is out

Reply:
what is the binary api for `tls_config` an array of flags?

Reply:
yes.. it currently supports --tls_config insecure=true ca_certs=path